web/src/portico/landing-page.js

```diff
@@ -135,6 +135,14 @@ $(() => {
         delete page_params.contributors;
         render_tabs(contributors);
     }
+
+    if (window.location.pathname === "/plans/" && window.location.hash === "#self-hosted") {
+        // Don't scroll to the target element
+        window.scroll({top: 0});
+        const $pricing_wrapper = $(".portico-pricing");
+        $pricing_wrapper.removeClass("showing-cloud");
+        $pricing_wrapper.addClass("showing-self-hosted");
+    }
```

Comment:
Just a small comment that in https://github.com/zulip/zulip/pull/27137, we're working towards just having each page have its own little JS file rather than these path-specific checks. Can be left for a follow-up when we get around to finishing that PR.